src/integrations/GoogleAds/browser.js

```diff
@@ -124,33 +128,26 @@ class GoogleAds {
   page(rudderElement) {
     logger.debug('in GoogleAdsAnalyticsManager page');
 
-    // Dynamic re-marketing is disabled
-    if (!this.dynamicRemarketing) {
-      const conversionData = this.getConversionData(
-        this.pageLoadConversions,
-        rudderElement.message.name,
-      );
-      if (conversionData.conversionLabel) {
-        const { conversionLabel } = conversionData;
-        const { eventName } = conversionData;
-        window.gtag('event', eventName, {
-          send_to: `${this.conversionId}/${conversionLabel}`,
-        });
-      }
-    } else {
-      const event = rudderElement.message.name;
-      if (!event) {
-        logger.error('Event name not present');
-        return;
-      }
-
-      let payload = {};
-      const sendToValue = this.conversionId;
+    const { name } = rudderElement.message;
+    const conversionData = this.getConversionData(this.clickEventConversions, name);
+    if (conversionData.conversionLabel && this.sendEvent('conversion', conversionData.eventName)) {
+      const { conversionLabel } = conversionData;
+      const { eventName } = conversionData;
+      window.gtag('event', eventName, {
+        send_to: `${this.conversionId}/${conversionLabel}`,
+      });
+    }
 
-      if (rudderElement.message.properties) {
-        payload = rudderElement.message.properties;
-      }
+    if (!name) {
+      logger.error('Event name not present');
+      return;
+    }
```

Comment:
Same as above shouldn't this check be happening on L130?

Reply:
can you check above comment